<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplaceDialog</class>
 <widget class="QDialog" name="FindReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>391</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>查找内容</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelFind">
        <property name="maximumSize">
         <size>
          <width>212</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>查找</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEditFind"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelReplace">
        <property name="text">
         <string>替换</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEditReplace"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnFindNext">
        <property name="text">
         <string>查找下一个</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnReplaceNext">
        <property name="text">
         <string>替换下一个</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btnReplaceAll">
        <property name="text">
         <string>替换全部</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>查找方法</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkReverse">
        <property name="text">
         <string>反向查找</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkCaseSesitively">
        <property name="text">
         <string>区分大小写</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkAllMatch">
        <property name="text">
         <string>全词匹配</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkRegrep">
        <property name="text">
         <string>正则匹配</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
